writer/starlark.go

```diff
@@ -162,6 +166,24 @@ func (sw *StarlarkWriter) writeBuffered() error {
 	return nil
 }
 
+// posArgs represents a list of literal positional argument and is written to support
+// the marshalling in WriteCommand.
+type posArgs []string
```

Comment:
Having an exported function return an unexported type is generally not recommended—particularly for a type with no real structure to protect. I'd advise just exporting the type:

```go
type ArgumentLiterals []string

func (a ArgumentLiterals) MarshalStarlark() ([]byte, error) { … }

// to use:
var args []string = …
sky, err := writer.ArgumentLiterals(args).MarshalStarlark()
```

It's true that this doesn't look like a function call when you construct one by hand, but I'm not sure that's really important.

Reply:
Done.